lambdas/log-redaction/src/redact-handler.ts

```diff
@@ -80,6 +57,81 @@ export class RedactHandler implements LambdaInterface {
     }
   }
 
+  private async saveRedactedLog(
+    redactLogGroup: string,
+    logStream: string,
+    logEvents: CloudWatchLogsDecodedData
+  ) {
+    logger.info("Putting redacted logs into " + redactLogGroup);
+
+    try {
+      const response: PutLogEventsCommandOutput = await cloudwatch.send(
+        new PutLogEventsCommand({
+          logGroupName: redactLogGroup,
+          logStreamName: logStream,
+          logEvents: logEvents.logEvents.map((event) => ({
+            id: event.id,
+            message: this.formatMessage(event.message),
+            timestamp: event.timestamp,
+            extractedFields: event.extractedFields,
+          })),
+        })
+      );
+      logger.info(JSON.stringify(response));
+    } catch (error) {
+      logger.error(`Error putting log events into ${redactLogGroup}: ${error}`);
+      throw error;
+    }
+  }
+
+  private async createLogStream(logStreamName: string, logGroupName: string) {
+    const query = await this.queryTable(logStreamName);
+
+    if (query.Count == 0) {
+      logger.info("Creating log stream " + logStreamName);
+
+      await cloudwatch.send(
+        new CreateLogStreamCommand({
+          logGroupName: logGroupName,
+          logStreamName: logStreamName,
+        })
+      );
+
+      await this.createLogStreamRecordInDB(logStreamName);
+    }
+  }
+
+  private async createLogStreamRecordInDB(logStream: string) {
```

Comment:
```suggestion
  private async saveLogStreamRecord(logStream: string) {
```
I think we use `save...` to say `store in a database` as a naming convention